hackclub.com.yaml

```diff
@@ -2596,6 +2596,11 @@ codecafe:
 - ttl: 1
   type: CNAME 
   value: cname.vercel-dns.com.
+  
+pizza:
+- ttl: 1
+  type: CNAME
+  value: hackclub.com/pizza.
```

Comment:
This is invalid CNAME formatting - CNAMEs don't take pathnames to redirect to since it happens at the DNS level and not the HTTP level. This should be done using a URL record instead.

Reply:
yeah you're right, we solved it eventually but forgot to make a pr to delete that